modelci/experimental/finetuner/trainer.py

```diff
@@ -137,12 +137,14 @@ def from_training_job(cls, training_job: TrainingJob) -> 'PyTorchTrainer':
     def start(self):
         def training_done_callback(future):
             model_train_curd.update(TrainingJobUpdate(_id=self._id, status=Status.PASS))
-            # TODO: save to database
+            # TODO: save to database and update model_status
             print(self.export_model())
 
         self._task = self._executor.submit(self.trainer_engine.fit, self.model, **self._data_loader_kwargs)
         self._task.add_done_callback(training_done_callback)
         model_train_curd.update(TrainingJobUpdate(_id=self._id, status=Status.RUNNING))
+        self.model_bo.model_status = ModelStatus.TRAINING
+        ModelService.update_model(self.model_bo)
```

Comment:
here you can create an empty model object, and add ID and model status to it. Then, you can the the `update_model` API.

Reply:
The original model is loaded, modified, and finally saved into the database with empty `dataset` and `metric` field as a new model, then finetuning process is then operated on this new model, I don't think an extra model object should be created here.

Reply:
As `ModelBO` object needs a few required parameters to create, maybe get the model object by id is more efficient?

Reply:
The speed is different, fetching an object from the database is considered a very slow operation compared to a memory object creation.
You can just create a model data object by setting the field to `None`, and set the id manually.
```
model_obj = ModelBO(name=None, framework=None, engine=None, version=None, dataset=None, metric=None, task=None, inputs=None, outputs=None)
model_obj._id = ...
```